401-sql-threat-detection-server-only/azuredeploy.json

```diff
@@ -0,0 +1,75 @@
+{
```

Comment:
let's combine these into a single sample - the deltas are pretty small and you can use the condition property to toggle features if needed.